src/Node/Stream.purs

```diff
@@ -61,6 +66,16 @@ type Writable r = Stream (write :: Write | r)
 -- | A duplex (readable _and_ writable stream)
 type Duplex = Stream (read :: Read, write :: Write)
 
+type Data = Either String Buffer
```

Comment:
What does this type synonym (and also `ChunkReader`) give us over writing out the real types? I generally tend to avoid type synonyms like these, as then there's more to keep in your head when you're reading the code.